src/pyOpenMS/pxds/LocalLinearMap.pxd

```diff
@@ -5,15 +5,29 @@ from Matrix cimport *
 cdef extern from "<OpenMS/ANALYSIS/PIP/LocalLinearMap.h>" namespace "OpenMS":
     
     cdef cppclass LocalLinearMap "OpenMS::LocalLinearMap":
+        # wrap-doc:
+            #   Trained Local Linear %Map (LLM) model for peak intensity prediction
```

Comment:
Has the "%Map" a deeper meaning or can that be renamed to just "Map"

Reply:
Fixed